scipy/stats/stats.py

```diff
@@ -5329,26 +5338,37 @@ def ttest_1samp(a, popmean, axis=0, nan_policy='propagate'):
 
     with np.errstate(divide='ignore', invalid='ignore'):
         t = np.divide(d, denom)
-    t, prob = _ttest_finish(df, t)
+    t, prob = _ttest_finish(df, t, alternative)
 
     return Ttest_1sampResult(t, prob)
 
 
-def _ttest_finish(df, t):
+def _ttest_finish(df, t, alternative):
```

Comment:
Just checking - how did you ensure that you changed all calls to `_ttest_finish`?

Reply:
I assumed it's a function private to stats.py (because of the _), so I just found all calling functions in stats.py and ensured they could pass through an alternative parameter.